src/peft/tuners/lora/config.py

```diff
@@ -259,6 +259,27 @@ class LoraConfig(PeftConfig):
             )
         },
     )
+    use_mora: bool = field(
+        default=False,
+        metadata={
+            "help": (
+                "Enable MoRA: High-Rank Updating for Parameter-Efﬁcient Fine-Tuning. This technique employs a square "
+                "matrix to achieve high-rank updating while maintaining the same number of trainable parameters. "
+                "To achieve it, MoRA introduces the corresponding non-parameter operators to reduce the input dimension "
+                "and increase the output dimension for the square matrix. Furthermore, these operators ensure that the "
+                "weight can be merged back into LLMs, which makes the method can be deployed like LoRA."
```

Comment:
Nice description. The last sentence is probably missing a word?

> which makes the method can be deployed like LoRA

Also, I would add that when using MoRA, IIUC the rank `r` is basically inferred such that the number of trainable parameters corresponds to that rank for LoRA.